Github-Issue-Guide.md

```diff
@@ -0,0 +1,55 @@
+# Filing an Issue on Oppia's GitHub: A Step-by-Step Guide for Non-Developers
+
+Welcome to Oppia's GitHub issue filing guide for non-developers! This step-by-step guide will help you navigate the process of creating a GitHub account and filing an issue on Oppia's GitHub repository. By following these instructions, you can contribute to improving Oppia without worrying about making mistakes. Let's get started!
```

Comment:
I think you can drop the "By following these ... making mistakes." sentence -- it actually might have the impact of increasing anxiety if making mistakes isn't something that the filer has considered previously :) I think the rest of the paragraph being encouraging helps with this already!

Reply:
Done